apps/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -35,23 +35,29 @@ import { GlobalScriptAction } from './scriptActions/GlobalScriptAction';
 import { Telemetry } from '../logic/Telemetry';
 import { AlreadyReportedError } from '../utilities/AlreadyReportedError';
 import { RushGlobalFolder } from '../api/RushGlobalFolder';
+import { NodeJsCompatibility } from '../logic/NodeJsCompatibility';
 
 /**
  * Options for `RushCommandLineParser`.
  */
 export interface IRushCommandLineParserOptions {
-  cwd?: string;   // Defaults to `cwd`
+  cwd: string;   // Defaults to `cwd`
+
+  /**
+   * @internal
```

Comment:
`@internal` has no meaning here, since `IRushCommandLineParserOptions` is private to this NPM package.